scripts/nginx/authelia.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#
+# Get our externial IP
+ex_ip="$(ip -br a | sed -n 2p | awk '{ print $3 }' | cut -f1 -d'/')"
```

Comment:
Maybe I missed this but I don't see this variable used here. Also, if it is, I'm curious how this would work for our non-static-IP friends